lms/envs/minimal.yml

```diff
@@ -20,6 +20,18 @@ MKTG_URL_LINK_MAP: {}
 MKTG_URL_OVERRIDES: {}
 REST_FRAMEWORK: {}
 
+# common.py uses `/edx/...` which we probably also don't want in production
+# systems so not sure if it's a good default but putting this here for now
+# since we know it won't work for local dev. The reason we wouldn't want it
+# in a production system is because we shouldn't save anythng to disk on a
+# production system and instead it should be going to some off-machine
+# storage so that we can more easily scale the system.
```

Comment:
I need help understanding this comment.
1. Is this all for `MEDIA_ROOT`, in addition to the clear `We need this because...` comment?
2. I don't follow the part about: `common.py uses /edx/...`. Are you simply stating that this default for MEDIA_ROOT` is not good for production? If so, can you just state that, and not compare to `common.py`, because I just find this comment long and confusing.
3. Additionally, I don't know what  `...putting this here for now since we know it won't work for local dev.` means? I would have guessed that this file is used for some sort of local dev, but not sure.
4. What is the `it` in: `The reason we wouldn't want it...`? Does `it` just refer to this default for `MEDIA_ROOT`?

The part I'm least clear on is the local dev comment, so you'd need to help reincorporate, but what do you think of something like the following:
```
# We need to default MEDIA_ROOT to a local path because otherwise 
#    ./lms/djangoapps/certificates/migrations/0003_data__default_modes.py will fail.
#   This is not a good default for a production system, because we should instead use
#   off-machine storage to more easily scale the system.
MEDIA_ROOT: "/tmp/edx-platform/media/"
```

Reply:
Makes sense, I'll update based on your suggestion.